tests/unit/validator/models/validationManager.spec.ts

```diff
@@ -83,22 +85,37 @@ describe('ValidationManager', () => {
         tilesetFilename: 'file.json',
         metadata: createMetadata(),
       };
-
       const result = validationManager.validateTilesetJson(payload);
-
       expect(result).toBe(`Unknown tileset name! The tileset file isn't founded!, tileset: ${payload.tilesetFilename}`);
     });
+
+    it('returns error string when the file is not a valid JSON', () => {
+      const payload: Payload = {
+        modelPath: createMountedModelPath('/Sphere'),
+        tilesetFilename: createInvalidTileset(),
+        metadata: createMetadata(),
+      };
+      const result = validationManager.validateTilesetJson(payload);
+      expect(result).toBe('Invalid tileset.json file!');
+    });
   });
 
   describe('#check footprint', () => {
-    it('returns true when the first and last coordinate are equal', () => {
+    it('returns true when the footPrint is valid and the first and last coordinate are equal', () => {
       const footprint = createFootprint();
 
       const result = validationManager.validateFootprint(footprint);
 
       expect(result).toBe(true);
     });
 
+    it('return "Invalid footprint!" when the footPrint is invalid', () => {
```

Comment:
Try to be more generic in your tests

```suggestion
    it('returns error string when the footPrint is invalid', () => {
```